usaspending_api/disaster/v2/views/spending_by_geography.py

```diff
@@ -272,7 +272,7 @@ def build_elasticsearch_result(self, response: dict) -> Dict[str, dict]:
                 state_fips = code_to_state.get(state_code, {}).get("fips", "")
                 for b in sub_bucket:
                     per_capita = None
-                    congress_code = b.get("key")
+                    congress_code = b.get("key")[2:]
```

Comment:
What change this?

Reply:
In `usaspending_api/etl/elasticsearch_loader_helpers/transform_data.py` I changed the elasticsearch fields from
```python 
        "pop_congressional_agg_key": lambda x: x["pop_congressional_code"],
        "pop_congressional_cur_agg_key": lambda x: x["pop_congressional_code_current"],
        "pop_county_agg_key": lambda x: x["pop_county_code"],
```
to 
```python
        "pop_congressional_agg_key": funcs.pop_congressional_agg_key,
        "pop_congressional_cur_agg_key": funcs.pop_congressional_cur_agg_key,
        "pop_county_agg_key": funcs.pop_county_agg_key,
```
This function adds the two character state abbr to the district/county code.  I made the change to follow the same pattern in awards as in transactions and to allow the reuse of the `build_elasticsearch_result` function in the view (in particular the `_key_to_geocode`) part of the function.
```
        def _key_to_geo_code(key):
            return f"{code_to_state[key[:2]]['fips']}{key[2:]}" if (key and key[:2] in code_to_state) else None
```
The old elastic search fields were being used in this view, so I just sliced off the first two state abbr characters to make it function as before.